tox.ini

```diff
@@ -121,10 +121,25 @@ commands =
 [testenv:build]
 description = Build a wheel and source distribution
 skip_install = True
-passenv = CFLAGS
+passenv = *
 deps =
     pep517
     twine
 commands =
-    python -m pep517.build -s -b {toxinidir} -o {toxinidir}/dist
+    python -m pep517.build {posargs: -s -b} {toxinidir} -o {toxinidir}/dist
```

Comment:
FWIW, `-s -b` is the default behavior, so you can omit it. `-s` excludes `-b` and `-b` excludes `-s`.

Reply:
[ref](https://github.com/pypa/pep517/blob/3f1949a727f879a997afeaa76ca10bbe2ef70011/pep517/build.py#L114-L117). I wish that were better documented.

Reply:
Nice, thanks!